src/RNCamera.js

```diff
@@ -58,6 +58,7 @@ type PropsType = ViewPropTypes & {
   autoFocus?: string | boolean | number,
   faceDetectionClassifications?: number,
   onFacesDetected?: ({ faces: Array<TrackedFaceFeature> }) => void,
+  onTextRecognized?: Function,
```

Comment:
can u improve this flowtype?

what is the shape of this function?

Reply:
Done